src/index.html

```diff
@@ -7,6 +7,359 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html" class="logo"><img src="images/logo.png" alt=""></a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">смартфоны и телефоны</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+            ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">гаджеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">планшеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">фото</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">видео</a>
+          </li>
+
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__cards">
+        <div class="product-card" data-qa="card">
+          <img class="product-card__img"
+            src="images/imac.jpeg"
+            alt="Моноблок Apple A1419"
+          >
+
+          <div class="product-card__description">
+            <h3 class="description-title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h3>
+
+            <div class="description-code">Код товара: 195434</div>
```

Comment:
```suggestion
            <p class="description-code">Код товара: 195434</p>
```
Тут теж краще було б використати семантичні теги